universal-login-example/src/relayer/startRelayer.js

```diff
@@ -1,7 +1,22 @@
 import TokenGrantingRelayer from './TokenGrantingRelayer';
+import knex from 'knex';
+import path from 'path';
 require('dotenv').config();
 const config = require('./config');
 
-const relayer = new TokenGrantingRelayer(config);
+const db = knex({
+  client: 'postgresql',
+  connection: {
+    database: 'universal_login_relayer_test',
+    user:     'postgres',
+    password: ''
+  },
+  migrations: {
+    directory: path.join(__dirname, '../../../universal-login-relayer/migrations'),
+    tableName: 'knex_migrations'
+  }
+});
```

Comment:
Should be taken from `knexfile.js`, from `development` section (should be developed not test database).

Reply:
✅